client/reader/tag-stream/controller.js

```diff
@@ -23,7 +23,7 @@ const exported = {
 				.toLowerCase()
 				.replace( /\s+/g, '-' )
 				.replace( /-{2,}/g, '-' ),
-			encodedTag = encodeURIComponent( tagSlug ).toLowerCase(),
+			encodedTag = tagSlug.toLowerCase(),
```

Comment:
Emojis have extra layer of encoding that makes everything just a tad bit more confusing.

I'm closing this for now